components/announcement_bar/no_internet_connection/no-internet-connection-svg.tsx

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+
+const NoInternetConnectionSvg: React.FC = () => {
+    return (
+        <svg
+            width='311'
+            height='216'
+            viewBox='0 0 311 216'
+            fill='none'
+            xmlns='http://www.w3.org/2000/svg'
+        >
+            <g clipPath='url(#clip0)'>
+                <path
+                    d='M61.9647 216C96.1868 216 123.929 213.897 123.929 211.302C123.929 208.707 96.1868 206.604 61.9647 206.604C27.7425 206.604 0 208.707 0 211.302C0 213.897 27.7425 216 61.9647 216Z'
+                    fill='#3D3C40'
+                    fillOpacity='0.08'
+                />
+                <path
+                    d='M47.3647 47.9266C58.1484 43.348 71.5797 46.2489 84.6968 48.2623C84.8189 45.8241 86.2984 42.8567 84.7486 41.1556C82.8656 39.089 83.039 36.9175 84.172 34.7336C87.0694 29.1489 82.9183 23.2065 78.7269 18.2655C77.7822 17.1543 76.5882 16.2813 75.242 15.7172C73.8958 15.1532 72.4353 14.9141 70.9792 15.0193L63.2149 15.5729C61.3256 15.7076 59.5225 16.4151 58.047 17.6008C56.5715 18.7864 55.4939 20.3936 54.9584 22.2072C53.1044 24.7242 52.1122 27.2343 52.7369 29.7312C49.96 31.6256 49.4934 33.9123 50.7413 36.4901C51.7994 38.1132 51.7881 39.6932 50.6931 41.2294C49.2551 43.188 48.1537 45.3718 47.4341 47.6915L47.3647 47.9266Z'
+                    fill='#152235'
```

Comment:
the decision to put the svg in memory with the component is because when the modal was being shown for the offline situation, the image was being shown as a broken-image. By putting it in memory we make sure that will be available in offline mode.

Reply:
Take a look at https://community-daily.mattermost.com/core/pl/jtiwrwc8mfff5ybc5tu5upq53w that shows how to make an svg theme compatible

Reply:
Spoke with Pablo today about this - putting the SVG into a component is fine, just need to fix up the theme styling so that it isn't jarring for dark themes

Reply:
@pablovelezvidal still need to handle the SVG themeing. Will approve once that's done

Reply:
If I understood correctly @alexsiclari the current design fits for light and dark themes

<img width="1411" alt="Captura de pantalla 2021-01-21 a las 19 32 23" src="https://user-images.githubusercontent.com/10082627/105547253-9b301400-5cfe-11eb-8efd-6bbf485004b9.png">
<img width="1494" alt="Captura de pantalla 2021-01-21 a las 19 32 50" src="https://user-images.githubusercontent.com/10082627/105547261-9e2b0480-5cfe-11eb-9911-6a2b10f1fd95.png">